src/main/scala/com/evolutiongaming/scaffeine/MetricsStatsCounter.scala

```diff
@@ -1,44 +1,57 @@
 package com.evolutiongaming.scaffeine
 
-import com.codahale.metrics.MetricRegistry
 import com.github.benmanes.caffeine.cache.stats.{CacheStats, StatsCounter}
-import nl.grons.metrics.scala.{MetricBuilder, MetricName}
+import io.prometheus.client.{CollectorRegistry, Counter}
 
 import scala.concurrent.duration.Duration
 
-class MetricsStatsCounter(registry: MetricRegistry, prefix: String) extends StatsCounter {
-  private val metrics = new MetricBuilder(MetricName(prefix), registry)
+class MetricsStatsCounter(registry: CollectorRegistry, prefix: String) extends StatsCounter {
 
-  private val misses = metrics.counter("misses")
+  private def createCounter(name: String): Counter = {
+    val c = Counter.build().name(s"${prefix}_$name".replaceAll("\\.", "_")).help(s"$prefix $name").create()
+    registry.register(c)
+    c
+  }
+
+//  private def createHistogram(name: String): Histogram = {
+//    val h = Histogram.build().linearBuckets(0, 100, 20).name(s"${prefix}_$name").help(s"$prefix $name").create() // 20 intervals by 100ms
+//    registry2.register(h)
+//    h
+//  }
+
+  private val misses = createCounter("misses")
 
-  private val hits = metrics.counter("hits")
+  private val hits = createCounter("hits")
 
-  private val eviction = metrics.counter("evictions")
+  private val eviction = createCounter("evictions")
 
-  private val failure = metrics.counter("load_failure")
+  private val failure = createCounter("load_failure")
 
-  private val success = metrics.counter("load_success")
+  private val success = createCounter("load_success")
 
-  private val time = metrics.timer("load_time")
+//  private val time = createHistogram("load_time")
```

Comment:
could you please remove this?

Reply:
sure, I will cleanup before/during merge